src/components/Main.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+// import { Route, Routes } from 'react-router-dom';
+import Navbar from './navbar/Navbar';
+import Cars from './cars/Cars';
+import './main.css';
+
+function Main() {
+  return (
+    <div className="main">
+      <Navbar />
+
+      {/* <Routes>
+        <Route exact path="/" element={<Cars />} />
+      </Routes> */}
```

Comment:
- [ ] Your project has several files which still have some old commented code.  Kindly remove all the code that was commented by you.

That way, the cleanliness of the repo will be improved 💪